lib/tasks/import.rake

```diff
@@ -5,9 +5,27 @@ require "gds_api/json_client"
 namespace :import do
   desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[organisation-slug, document-type]"
   task :whitehall_migration, %i[organisation_slug document_type] => :environment do |_, args|
-    WhitehallMigration.create!(organisation_slug: args.organisation_slug,
-                               document_type: args.document_type,
-                               start_time: Time.zone.now)
+    ActiveRecord::Base.transaction do
+      WhitehallMigration.create!(organisation_slug: args.organisation_slug,
+                                 document_type: args.document_type,
+                                 start_time: Time.zone.now)
+
+      items_per_page = 100
+
+      1.step do |page_number|
+        puts "Loading page #{page_number}"
+        params = {
+          lead_organisation: args.organisation_slug,
+          type: args.document_type,
+          page_number: page_number,
+          page_count: items_per_page,
+        }
+        endpoint = "/government/admin/export/document?#{params.to_query}"
+        whitehall_export = WhitehallImporter.whitehall_get_json(endpoint)
+        WhitehallImporter.load_index_page(whitehall_export)
+        break if whitehall_export["page_count"] < items_per_page
```

Comment:
I changed this to use `until` then the linter complained:

```
Lint/Loop: Use Kernel#loop with break rather than begin/end/until(or while). (https://rubystyle.guide#loop-with-break)
```